test/test_spell_check.py

```diff
@@ -25,8 +25,7 @@ def test_spell_check():
             '--enable=spelling',
             '--spelling-dict=en_US',
             '--ignore-comments=no',
-            '--spelling-private-dict-file=' +
-            str(spell_check_words_path),
+            '--spelling-private-dict-file=' + str(spell_check_words_path),
```

Comment:
nit: I'd use `f'--spelling-private-dict-file={spell_check_words_path}'.
It's both more readable and faster than string addition